notebooks/text_classification/solutions/word2vec.ipynb

```diff
@@ -0,0 +1,1453 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # and vocabulary size.
</pre><p>Add that as a docstring instead</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Good catch! </p>